extensions/sample-ui-plugin/src/client/publish/main.tsx

```diff
@@ -72,6 +77,14 @@ export const Main: React.FC<{ title: string }> = (props) => {
           onChange={updateVal3}
         ></input>
       </div>
+      <div className={buttonBar}>
+        <button className={backButton} onClick={onBack}>
+          Back
+        </button>
+        <button className={saveButton} onClick={onSave} disabled={!configIsValid}>
```

Comment:
onClickSaveButton or onClickSave?. Usually, we can follow the pattern of on{EventName}{Action}